pac4j-jwt/pom.xml

```diff
@@ -33,6 +33,10 @@
 			<artifactId>pac4j-core</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>commons-codec</groupId>
```

Comment:
Yes, except that we might need to check to see if the message is base-64 encoded before we can decode it. This is related to the API change we are discussing below.